dropwizard/service/src/test/java/org/apache/polaris/service/dropwizard/it/QuarkusRestCatalogIntegrationTest.java

```diff
@@ -18,6 +18,8 @@
  */
 package org.apache.polaris.service.dropwizard.it;
 
+import io.quarkus.test.junit.QuarkusTest;
 import org.apache.polaris.service.it.test.PolarisRestCatalogIntegrationTest;
 
-public class DropwizardRestCatalogIntegrationTest extends PolarisRestCatalogIntegrationTest {}
+@QuarkusTest
```

Comment:
nit: Do you think it should be `@QuarkusIntegrationTest` in the spirit of backbox tests?

Reply:
Yes, but later: we don't have an `intTest` configuration in polaris yet :-)

Reply:
Just add this to your `build.gradle.kts` (after #714)

```kotlin
testing {
  suites {
    val intTest by registering(JvmTestSuite::class)
  }
}

tasks.named("check") { dependsOn(testing.suites.named("intTest")) }
```

Reply:
Nice, let's do it in a follow-up PR after the renamings etc.

Reply:
no rush :)